src/ui/server/controllers/insurance/policy/single-contract-policy/total-contract-value/index.ts

```diff
@@ -45,9 +45,15 @@ export const pageVariables = (referenceNumber: number, currencies: Array<Currenc
   const currency = getCurrencyByCode(currencies, policyCurrencyCode);
 
   return {
-    FIELD: {
-      ID: FIELD_ID,
-      ...FIELDS.CONTRACT_POLICY.SINGLE[FIELD_ID],
+    FIELDS: {
+      TOTAL_CONTRACT_VALUE: {
+        ID: TOTAL_CONTRACT_VALUE,
+        ...FIELDS.CONTRACT_POLICY.SINGLE[TOTAL_CONTRACT_VALUE],
```

Comment:
is it worth dereferencing?

Reply:
We could - personally I think this is clearer, because it's a `FIELDS` object that is then using another imported thing called `FIELDS`, destructuring the imported `FIELDS` would make it obscured IMO.

We have this approach in most places, should be done separately if we do want make this change.